applications/GeoMechanicsApplication/tests/cpp_tests/test_interface_element_utilities.cpp

```diff
@@ -0,0 +1,259 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_geometries/line_interface_geometry.h"
+#include "custom_utilities/interface_element_utilities.h"
+#include "geo_mechanics_fast_suite.h"
+
+#include <boost/numeric/ublas/assignment.hpp>
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(InterfaceElementUtilities_RotationMatrixForHorizontalInterfaceIsIdentityMatrix,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    PointerVector<Node> nodes;
+    nodes.push_back(Kratos::make_intrusive<Node>(1, 0.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(2, 5.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(3, 0.0, 0.0, 0.0));
+    nodes.push_back(Kratos::make_intrusive<Node>(4, 5.0, 0.0, 0.0));
+    const LineInterfaceGeometry geometry(1, nodes);
+
+    // Act
+    const auto local_coordinate = array_1d<double, 3>{0.0, 0.0, 0.0};
```

Comment:
If anything needs comment here, it is that the local_coordinate has 2 extra unused components.